core/crypto/sha/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+add_library(filecoin_sha INTERFACE)
```

Comment:
Declared, but not used

Reply:
It is an interface library, there is no need to directly include it.